lib/rubocop/cop/style/redundant_self.rb

```diff
@@ -176,7 +176,7 @@ def regular_method_call?(node)
             node.implicit_call?)
         end
 
-        def on_argument(node)
+        def handle_argument(node)
```

Comment:
There is a `argument` node group, so this conflicts. This is the only place in rubocop where this happens but it is worth pointing this out. Unfortunately, this cop crashses without changing this so I'm not sure if this change is actually possible to do.

Would love other thoughts on this.